docs/UserGuide.md

```diff
@@ -92,30 +96,28 @@ Here are the potential error messages that you may receive and here's how to fix
 | `Invalid command format!` | The command you input does not follow the specified format | Ensure the command you entered follows the following format: `add n/NAME p/PHONE_NUMBER e/EMAIL_ADDRESS a/HOME_ADDRESS [t/TAG]...`, replacing `NAME` with employee name, `PHONE_NUMBER` with employee phone number, `EMAIL_ADDRESS` with employee email address, `HOME_ADDRESS` with employee home address. `[t/TAG]...` is optional, with `[t/TAG]` representing one or more tags and `TAG` being the tag name.
 | `FIELD should FORMAT` where `FIELD` is an input like `Names` or `Phone numbers` and `FORMAT` contains additional information about the field's format. | The input does not follow the format prescribed. For example, the entered phone number might contain alphabets. | Follow the on screen message to fix the field in question. For example, `Phone numbers should only contain numbers, and it should be at least 3 digits long` means that the input phone number does not follow the prescribed format. |
-| `This employee already exists in the address book` | The provided employee name is already found in HRMate | Use another name for the employee. For example, if trying to add another "John Doe", use the name "John Doe (HR)" to differentiate between the existing John Doe. HRMate does this name checking to prevent unintentional duplicate employee entries. |
+| `This employee already exists in the address book` | The provided employee name is already found in HRMate | Use another name for the employee. For example, if trying to add another "John Doe", use the name "John Doe (HR)" to differentiate between the existing John Doe. HRMate does this name-checking to prevent unintentional duplicate employee entries. |
 
+#### Listing all employees
+When you would like to view all employees, you can use the `list` [command](#glossary) to view them.
 ### Listing all employees: `list`
 
-* **What It Does:**
-  * Shows a list of all employees in HRMate
-* **Format:**
-  * `list`
-* **Examples:**
+1. Type in the following [command](#glossary) in the [command box](#glossary): `list`.
+2. Press "enter" on your keyboard and you should see the information of all employees listed in the [employee list](#glossary).
 
-| S/N | Command information                                                                        |
-|-----|--------------------------------------------------------------------------------------------|
-| 1   | **Command:** `list` <br><br> **Output:** Shows a list of all employees in the address book |
-* **Acceptable Values:**
-  * This command has no parameters
-* **Expected Output on Success:**
-  * GUI Changes: A list of employees is displayed in a dedicated interface section
-* **Expected Output on Failure:**
-  * N/A (no expected failure)
+> [!NOTE]  
+> This command does not require any parameters. However, to account for possible typing mistakes, the program reads in inputs like `list 123`, `list abc`, and `list 1a2b` all as the command `list`.
```

Comment:
would putting "HRMate reads in inputs like..." instead of "the program reads in inputs like..." sound more friendly?